src/reanimated2/core.js

```diff
@@ -56,11 +17,6 @@ export function requestFrame(frame) {
   }
 }
 
-global._WORKLET = false;
-global._log = function(s) {
-  console.log(s);
-};
-
```

Comment:
All "globals" moved into `globals.js`. These are available in test env and don't need to be mocked.

Reply:
I'm not familiar with plugin, @karol-bisztyga is this safe to do? (I'm mostly not sure if _WORKLET is used per module or not)

Reply:
It was redeclare agains, several lines bellow. I believe this served as test mock and is a leftover, somehow.